vendor/diff-parsers-ext-parsewyc.patch

```diff
@@ -0,0 +1,953 @@
++    (*| LBRACKET pattern_semi_list error
++      { unclosed "[" $loc($1) "]" $loc($3) }*)
+     | LBRACKETBAR pattern_semi_list BARRBRACKET
+       { Ppat_array $2 }
+     | LBRACKETBAR BARRBRACKET
+       { Ppat_array [] }
+-    | LBRACKETBAR pattern_semi_list error
+-      { unclosed "[|" $loc($1) "|]" $loc($3) }
++    (*| LBRACKETBAR pattern_semi_list error
++      { unclosed "[|" $loc($1) "|]" $loc($3) }*)
+   ) { $1 }
+ 
+ pattern_comma_list(self):
+     pattern_comma_list(self) COMMA pattern      { $3 :: $1 }
+   | self COMMA pattern                          { [$3; $1] }
+-  | self COMMA error                            { expecting $loc($3) "pattern" }
++  (*| self COMMA error                            { expecting $loc($3) "pattern" }*)
+ ;
+ %inline pattern_semi_list:
+   ps = separated_or_terminated_nonempty_list(SEMI, pattern)
+     { ps }
+ ;
+@@@@
+   | PLUS                                    { Covariant, NoInjectivity }
+   | MINUS                                   { Contravariant, NoInjectivity }
+   | BANG                                    { NoVariance, Injective }
+   | PLUS BANG | BANG PLUS                   { Covariant, Injective }
+   | MINUS BANG | BANG MINUS                 { Contravariant, Injective }
+-  | INFIXOP2
++  (*| INFIXOP2
+       { if $1 = "+!" then Covariant, Injective else
+         if $1 = "-!" then Contravariant, Injective else
+-        expecting $loc($1) "type_variance" }
+-  | PREFIXOP
++        expecting $loc($1) "type_variance" }*)
++  (*| PREFIXOP
+       { if $1 = "!+" then Covariant, Injective else
+         if $1 = "!-" then Contravariant, Injective else
+-        expecting $loc($1) "type_variance" }
++        expecting $loc($1) "type_variance" }*)
+ ;
+ 
+ (* A sequence of constructor declarations is either a single BAR, which
+    means that the list is empty, or a nonempty BAR-separated list of
+    declarations, with an optional leading BAR. *)
+@@@@
+       { [head], Closed }
+   | head = field
+   | head = inherit_field
+       { [head], Closed }
+   | DOTDOT
+-      { [], Open }
++      { [], (Open : closed_flag) }
+ ;
+ %inline field:
+   mkrhs(label) COLON poly_type_no_attr attributes
+     { let info = symbol_info $endpos in
+       let attrs = add_info_attrs info $4 in
+@@@@
+     UIDENT                    { $1 }
+   | LIDENT                    { $1 }
+ ;
+ val_extra_ident:
+   | LPAREN operator RPAREN    { $2 }
+-  | LPAREN operator error     { unclosed "(" $loc($1) ")" $loc($3) }
+-  | LPAREN error              { expecting $loc($2) "operator" }
+-  | LPAREN MODULE error       { expecting $loc($3) "module-expr" }
++  (*| LPAREN operator error     { unclosed "(" $loc($1) ")" $loc($3) }*)
++  (*| LPAREN error              { expecting $loc($2) "operator" }*)
++  (*| LPAREN MODULE error       { expecting $loc($3) "module-expr" }*)
+ ;
+ val_ident:
+     LIDENT                    { $1 }
+-  | val_extra_ident           { $1 }
++  (*| val_extra_ident           { $1 }*)
```

Comment:
Why this diff ?

Reply:
It was a mistake, I've uncommented the line